flows.yaml

```diff
@@ -3436,6 +3446,7 @@ integrations:
                 amount: number
                 vatCode?: string
                 description?: string
+                locationId?: string
```

Comment:
why optional if it is supposed to be mandatory?

Reply:
Yeah...

I made it required at first but that same object is also used by the `credit-notes` `NetsuiteCreditNote` (lines) where it is not required which broke the compilation so I made it optional but then required by the zod override check. Not pretty I know but I didn't want to wrangle the types too much to make it work.

Reply:
if credit note and invoice lines are diverging I would be tempted to have different model/types.

Reply:
Yeah, fair. I'll open up an issue to improve this at a later date. Thanks

Reply:
Tracking in https://linear.app/nango/issue/NAN-1866/pr-followup